### ISSUE TITLE
Finkelstein sign abnormal

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10754]
**Preferred term label:**
Finkelstein sign abnormal

**Synonyms**
Radial sided wrist pain worsens with passive wrist ulnar deviation with thumb grasped in palm

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/books/NBK539768/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
De Quervain tenosynovitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001396
- label: Positive Finkelstein sign
- parents: [Abnormal wrist physiology (HP:0025799)](https://hpo.jax.org/app/browse/term/HP:0025799)
- pmid: PMID:30969590
- def: Finkelstein's test produces severe tenderness and usually pain on the radial aspect of the wrist when the thumb is flexed into the palm and the wrist is ulnar deviated.
- comment: The Finkelstein test is a provocative test for diagnosis of De Quervain disease that can easily be performed in an office setting or at the bedside.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10754
- synonyms:
